spel/minimal-linux.json

```diff
@@ -247,7 +327,7 @@
                 "SPEL_AMIGENSOURCE={{ user `spel_amigen7source` }}",
                 "SPEL_AMIGENSTORLAY={{ user `spel_amigen7storlay` }}",
                 "SPEL_AMIUTILSSOURCE={{ user `spel_amiutilsource` }}",
-                "SPEL_AWSCLISOURCE={{ user `spel_awsclisource` }}",
+                "SPEL_AWSCLISOURCE={{ user `aws_cliv1_source` }}",
```

Comment:
Ah... I'd assumed "just `aws` because it's specific to the AWS builder rather than spel more-broadly (i.e., it wouldn't exist in Azure builders).

Reply:
Yeah, mean "builder" in the packer-sense, where we expose the options used in the packer builder. Just trying to avoid renaming things that packer already has names for, and everything else that is specifically about spel gets the `spel_` prefix...

Reply:
Ah... No worries. I misinterpreted the implied-scope of the names.

Reply:
Resolved in a2ddd0901f255c882c2c9510c930302e468b2fed